static/api/extensions/redhat/openshift-local/2.1.3/README.md

```
@@ -0,0 +1,162 @@
+# Red Hat OpenShift Local extension
+
+Integration for [Red Hat OpenShift Local][product page] clusters. It will help you install and set up the environment, and allows you to control the lifecycle and configuration from [Podman Desktop][podman-desktop]. Some of the features available in the extension include:
+  * Start/Stop/Restart/Delete OpenShift Local clusters
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)